src/types/model.ts

```diff
@@ -372,12 +372,19 @@ export interface DataAccessRequest {
   intellectualPropertySummary: string;
   publications: Array<Publication>;
   presentations: Array<Presentation>;
-  dataManagementIncident: DataManagementIncident;
+  dmi: DataManagementIncident;
   researchPlans: string;
-  closeOutSupplement: 'PROJECT_COMPLETED' | 'REQUESTOR_MOVED_INSTITUTION' | 'PROJECT_TRANSFERRED' | 'PROJECT_SUPERSEDED';
+  closeoutSupplement: CloseOutSupplement;
```

Comment:
Capitalization and exact name matters when mapping to the db model. The changes to dmi and closeoutSupplement match the state in consent.